### PR TITLE
feat(ansible): update environmental setup

### PR DESCRIPTION
**Pull request by manato**
## Related Links
- https://github.com/tier4/edge-auto/pull/4

## Description

Update ansible roles to set up edge.auto_jetson environment.

## Review Procedure

Please run `setup-dev-env.sh` to set up the environment.

## Remarks

This PR contains the `netplan` role, which fixes the IP address for `eth1`.  If you are using this network interface for ssh connection, running `ansilbe-playbook` may cause ssh disconnection.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR**: Required to pass before the merge.

### CODE REVIEW

**Comment by aohsato** [comment]
@manato How about PR status?

**Comment by manato** [comment]
@aohsato As well as other PR, I'll open this PR because I guess it's ready.
Please note that I set the default camera configuration to be:
- FSYNC trigger mode
- fixed exposure time (11ms)

in this PR.

**Comment by aohsato** [comment]
> Please note that I set the default camera configuration to be...

@manato Could you add this explain to our tutorials? https://github.com/tier4/edge-auto/pull/3

File: ansible/setup.yaml [hunk 1]
```diff
@@ -16,3 +21,6 @@
     - role: tier4_hdr_camera_driver
       when: prompt_install_camera_driver == 'y'
     - role: ros2
+    - role: ptp4l_client
+    - role: enlarge_txqueue
+    - role: netplan
```
Comment (aohsato) [inline ansible/setup.yaml:26]: [imo] It's better that user can choose netplan installation or not.

Comment (aohsato) [reply ansible/setup.yaml:26]: or whole network settings

Comment (aohsato) [reply ansible/setup.yaml:33]: nice! 6577009a4b4dfc109219c3a2e5dc598328c80999

**Review by aohsato — APPROVED** [review]
Working on my environment, also good work.

**Pull request merged**